<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkGame</class>
 <widget class="QMainWindow" name="NetworkGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="infoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>250</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currTurnLabel">
       <property name="text">
        <string>当前是第 0 回合</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="myPlayerLabel">
          <property name="text">
           <string>我方：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currentPlayerLabel">
          <property name="text">
           <string>当前回合玩家：</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="myTimeoutCntLabel">
          <property name="text">
           <string>我方超时次数：</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="admitDefeatButton">
       <property name="text">
        <string>认输</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="gameRuleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>400</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>【提示】
自动判负的情形：
   ·超时3次；
   ·在第20、25、30回合结束时，离开大本营棋子数目不足5、8、10颗；
   ·如果双方在同一回合触发上述条件， 自动判平局。</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
